salon_creator/lib/screens/salon_creation_screen.dart

```diff
@@ -8,20 +20,529 @@ class SalonCreationScreen extends StatefulWidget {
 }
 
 class _SalonCreationScreenState extends State<SalonCreationScreen> {
+  bool isFilled = false;
+  bool inProgress = false;
+  final ImagePicker imagePicker = ImagePicker();
+  int index = 0;
+  List<XFile> _imageFiles = [];
+
+  ScrollController _scrollController = ScrollController();
+  TextEditingController categoriesController = TextEditingController();
+  TextEditingController descriptionController = TextEditingController();
+  TextEditingController priceController = TextEditingController();
+  TextEditingController salonNameController = TextEditingController();
+  XFile image;
+
+  void _updateScreenContext() {
+    var _hasProfileChanged = salonNameController.text.isNotEmpty &&
```

Comment:
I think these checks have to be OR checks. Profile is considered change if ANY of these become true.

Reply:
Do you mean that in salon creation users should be able to finish their salon creation even all of the forms are not filled?